package/MDAnalysis/core/universe.py

```diff
@@ -313,10 +326,17 @@ class Universe(object):
         Universe now can be (un)pickled.
         ``topology`` and ``trajectory`` are reserved upon unpickle.
 
+    .. versionchanged:: 2.4.0
+      added guess_TopologyAttributes API
+      guessing masses and atom types when topology is read from a registered parser
+
+
     """
+
     def __init__(self, topology=None, *coordinates, all_coordinates=False,
                  format=None, topology_format=None, transformations=None,
-                 guess_bonds=False, vdwradii=None, in_memory=False,
+                 guess_bonds=False, vdwradii=None, context='default',
+                 to_guess=(), force_guess=(), in_memory=False,
```

Comment:
```suggestion
                 to_guess=('mass', 'type'), force_guess=(), in_memory=False,
```

By default, we guess masses and types. This is where it should appear.